### ISSUE TITLE
Vagrant up box store pool template

### ISSUE THREAD

**Issue by SaimonL** [vagrant-libvirt/vagrant-libvirt#538]
Hi I have 3 storage pools. First store pool called "default", the second one is called "node-1" and the third on is called 'node-2'. When I do "vagrant up" for the first time I want vagrant to download the box to "default" pool first then create the vagrant instance image on "node-1" pool. I have tried:

``` ruby
config.vm.provider :libvirt do |libvirt|
  libvirt.storage_pool_name = 'node-1'
end
```

What ends up happening is both the downloaded template and the vagrant image both ends up on 'node-1' storage pool. Then for my second vagrant instance I have this configuration:

``` ruby
config.vm.provider :libvirt do |libvirt|
  libvirt.storage_pool_name = 'node-2'
end
```

Now vagrant pulls the box template from 'node-1' store pool. The purpose of the 'node' storage pool is to be able to eject then as needed after shutting down all vagrant instance on those storage pools. Since all the base vagrant templates are 'node-1' I can't do that.

So my question is how can I have the first vagrant template be on a different storage pool and the vagrant instance in another storage pool.

Also I have tried this as well

``` ruby
    libvirt.storage :file,
      size: '20G',
      type: 'qcow2',
      path: '/mnt/node-1'
```

All it does is adds a second hard drive to the vagrant instance.

**Comment by infernix** [comment]
I think I know what you are saying, but can you provide a full `Vagrantfile` example I can reproduce with?

**Comment by electrofelix** [comment]
@SaimonL can you have a look at https://github.com/abbbi/vagrant-libvirt/commit/015c7978b6a507c1d18b3faff17a3262b51f1f6a and see if it aligns with what you are looking for?

Probably will close this as the duplicate as the newer issue contains references to the commit needed.

**Comment by electrofelix** [comment]
@SaimonL going to close this as covered by #952 and likely resolved by #953, though expect #974 will also be needed to avoid surprised. But that first change should mean that you configure `storage_pool_name` to be the same for each node and set `snapshot_pool_name` instead. May also want to discuss about the naming of `snapshot` to describe the live volume storage as that may get confused with the idea of vagrant snapshots.
